src/ArcCommander/ArgumentProcessing.fs

```diff
@@ -279,18 +279,24 @@ module ArgumentProcessing =
             let serializeF = serializeXSLXWriterOutput writeF >> sprintf "%s\n\n%s" header
 
             let deserializeF (s : string) : 'A =
-                s.Replace(sprintf "%s\n\n" header, "").Split '\n'
-                |> Seq.map (fun x ->                 
-                    match splitAtFirst ':' x with
-                    | k, Field v ->
-                        ISADotNet.XLSX.SparseRow.fromValues [k;v]
-                    | _ -> failwith "ERROR: File was corrupted in Editor"
+                s.Split('\n')
```

Comment:
I could but I would strongly suggest to let in there because otherwise the function fails.

This new bug was unknowingly induced when fixing the bug for Windows Enterprise versions where `\n` would not correctly translate into a line break and thus had to be replaced with `System.Environment.NewLine`. Unfortunately, this now causes a new bug where the `'\037'` char (Carriage Return) elicits problems with matching.

Reply:
Okay could you then maybe open an issue describing the problem in detail?